lib/authorization.rb

```diff
@@ -58,14 +58,13 @@ def self.matched_policies(subject_id, actions = nil, object_id = nil)
 
     DB[<<~SQL, {subject_id: subject_id, actions_filter: actions_filter, object_filter: object_filter}].all
       SELECT object_applied_tags.tagged_id, object_applied_tags.tagged_table, subjects, actions, objects
-      FROM accounts AS subject
-        JOIN applied_tag AS subject_applied_tags ON subject.id = subject_applied_tags.tagged_id
+      FROM applied_tag AS subject_applied_tags
```

Comment:
Good improvements. The initial idea was using more resources as subject. API tokens are good example. Additionally, you can authorize virtual machines to access other resources on AWS, which is something we might consider doing in the future.